src/navigations/HomeBottomTabs.js

```diff
@@ -280,7 +280,7 @@ function HomeBottomTabs({navigation}) {
   };
 
   return (
-    <View style={styles.container}>
+    <SafeAreaView style={styles.container}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `View` to `SafeAreaView` is a good one. It ensures that the content doesn't overlap with system UI elements on iOS devices, such as the notch or home indicator.



<!-- This is an auto-generated comment by OSS CodeRabbit -->